src/arrayMethodReduce.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function (callback, initialValue) {
+    let startIndex = 0;
+    let prev = initialValue;
```

Comment:
```suggestion
    let acc = initialValue;
```

Reply:
it's accumulator, not just prev value

Reply:
Yes, agree with you. You write this one on the previous tasks, will change this soon